core/api/src/app/accounts/get-account-transactions-for-contact.ts

```diff
@@ -8,29 +9,50 @@ import { WalletsRepository } from "@/services/mongoose"
 export const getAccountTransactionsForContact = async ({
   account,
   contactUsername,
-  paginationArgs,
+  rawPaginationArgs,
 }: {
   account: Account
   contactUsername: Username
-  paginationArgs?: PaginationArgs
-}): Promise<PaginatedArray<WalletTransaction> | ApplicationError> => {
+  rawPaginationArgs: {
+    first?: number | null
```

Comment:
the type here are `number | undefined | null`. any reason to not use undefined?

```suggestion
    first?: number | undefined
```

Reply:
The arguments are technically both nullable and undefinable from the gql layer but it looks like that is actually the case in many places and we still just use undefined. I will switch to that.